### ISSUE TITLE
WandbLogger : How to visualize gradients and values of model 

### ISSUE THREAD

**Issue by makinzm** [PyTorchLightning/pytorch-lightning#11134]
### To Reproduce

I try [a demo](https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.loggers.wandb.html#pytorch_lightning.loggers.wandb.WandbLogger.log_table) which your teams provide in order to understand how to use WandbLogger

- [This is my Wandb Project](https://wandb.ai/nozomi_maki/MNIST-1218?workspace=user-nozomi_maki)
- [This is my code](https://github.com/makinzm/UNDONE_local/blob/main/For_share_Optimize_Pytorch_Lightning_models_with_Weights_%26_Biases.ipynb)

However, I don't watch gradients or values of model.

### Expected behavior

Your teams explained that I can watch gradients like, 
![image](https://user-images.githubusercontent.com/40193664/146569283-1350a088-362e-41ea-ba3a-c6e546bc9cf0.png).
However, I couldn't understand how to visualize this. (Here is [my wandb project](https://wandb.ai/nozomi_maki/MNIST-1218?workspace=user-nozomi_maki))

### Environment
- Google Colaboratory : (I didn't check all package version).

### Additional context

- If you know how to visualize not only gradients but also values, I would appreciate it.

**Comment by scottire** [comment]
Hey! Glad to hear you are trying out W&B. 

To see the gradients you need to click into the run, rather than looking at your workspace (where _all_ of your runs are).

To go to the page for a given run, you just click the run name on the left of your workspace.
![Screenshot 2021-12-17 at 16 11 36](https://user-images.githubusercontent.com/8557070/146574402-066b96a9-e6b2-4f09-a6c4-e3bf7cd8d8f2.png)

Here's your run page:
[https://wandb.ai/nozomi_maki/MNIST-1218/runs/3b4dbk8f](https://wandb.ai/nozomi_maki/MNIST-1218/runs/3b4dbk8f?workspace=user-_scott
)

**Comment by borisdayma** [comment]
You can also customize the log frequency of your gradients and also log parameters.

See reference documentation: https://docs.wandb.ai/ref/python/watch

I'll add some of these details in PL documentation built from the docstring.

**Comment by makinzm** [comment]
Thank you for teaching me.
I can have checked the gradients of my model

![image](https://user-images.githubusercontent.com/40193664/146626703-56938a26-c32f-4384-a8d2-60efaa82a7e2.png)

and check values via csv.

![image](https://user-images.githubusercontent.com/40193664/146626752-85a17351-d6c0-49b8-871b-951d05523b3f.png)


However, I cannot check the values of parameters like as gradient.
I would like to visualize each epoch shows like this : 

```
for i in model.state_dict().items():
    print(i[0]) # model.conv1.weight
    break
print(i[1].reshape(i[1].shape[0],-1).shape) # torch.Size([32, 9])
plt.hist(i[1].reshape(i[1].shape[0],-1),color=colors[:i[1].shape[0]],width=0.2,alpha=0.5)
plt.show()
```
![image](https://user-images.githubusercontent.com/40193664/146627707-79bdc919-4d90-4918-a119-36b21ff1b27d.png)

If you teach me , I appreciate it.
